docs/workflows/snapshotting.md

```diff
@@ -21,7 +21,7 @@ A cron job is run every hour that introspects at least 1 out every 24 repositori
 
 1. Snapshots can be used for patching a RHEL client by configuring a client to use a snapshot URL.  The easiest way to do this is by downloading the configuration file from  `/api/content-sources/v1/snapshots/{snapshot_uuid}/config.repo` and placing the file inside the /etc/yum.repos.d/ directory on the client.
```

Comment:
```suggestion
1. Snapshots can be used for patching a RHEL client by configuring a client to use a snapshot URL.  You can do this is by downloading the configuration file from  `/api/content-sources/v1/snapshots/{snapshot_uuid}/config.repo` and placing the file inside the /etc/yum.repos.d/ directory on the client.
```

Reply:
You are not supposed to make judgements on how easy, simple, or quick something is in tech docs.

Reply:
@swadeley @jlsherrill added the snapshotting docs a bit ago, i added docs for templates (docs/workflows/templates.md) in this PR. i can adjust this wording though if it's preferable?

Reply:
Please do